src/app.css

```
@@ -36,32 +50,60 @@
+	--color-neutral-50: #0b0d10;
+	--color-neutral-100: #282f36;
+	--color-neutral-200: #3d454d;
+	--color-neutral-300: #5e6670;
+	--color-neutral-400: #7c838b;
+	--color-neutral-500: #929891;
+	--color-neutral-600: #d3d5d0;
+	--color-neutral-700: #e7e7e1;
+	--color-neutral-800: #f1f2f0;
+	--color-neutral-900: #f3f3ef;
+	--color-neutral-950: #fafaf8;
 	color-scheme: light;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)